code/items/weapons/TTTWeapon.cs

```diff
@@ -157,6 +157,24 @@ public override void Simulate(Client owner)
                 }
             }
 
+            //No idea where else to put this. So it goes here.
+            if (Input.Released(InputButton.View) && AmmoClip > 0) //Todo: Move this to be a bindable key (default Z???)
+            {
+                if (IsServer)
+                {
+                    var ammoBox = new BuckshotAmmo()
+                    {
+                        AmmoAmount = AmmoClip,
+                        Position = Owner.EyePos + Owner.EyeRot.Forward * 50,
```

Comment:
Can we change the `50` to a const static int at the top of the file? and give it an appropriate name.